src/app/debug/table/table-settings-modal/table-settings-modal.component.html

```diff
@@ -26,7 +26,7 @@ <h5>Debug tree</h5>
         <h5>Editor</h5>
         <div class="checkbox " title="When checked, search window is shown when editor is loaded">
           <label for="show-search-on-load">Show search window in editor on load</label>
-          <input class="btn btn-info my-2 p-1 w-fit" data-cy-show-search-on-load id="show-search-on-load"
+          <input class="btn btn-info my-2 p-1 w-fit" data-cy-settings-modal="showSearchOnLoad" id="show-search-on-load"
```

Comment:
Modal mag je hier weglaten
```suggestion
          <input class="btn btn-info my-2 p-1 w-fit" data-cy-settings="showSearchOnLoad" id="show-search-on-load"
```